android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -526,6 +528,23 @@ public boolean handleMessage(Message msg)
 				Activity activity = getWindowActivity();
 				if (activity != null) {
 					activity.setTitle(TiConvert.toString((Object) (msg.obj), ""));
+
+					if (windowActivity != null && windowActivity.get() != null
+						&& windowActivity.get().getSupportActionBar() != null) {
+						ActionBar actionBar = windowActivity.get().getSupportActionBar();
+						if (actionBar.getTitle() instanceof SpannableStringBuilder) {
+							SpannableStringBuilder ssb;
+							ssb = (SpannableStringBuilder) actionBar.getTitle();
+							ssb = new SpannableStringBuilder(TiConvert.toString((Object) (msg.obj), ""));
```

Comment:
You are re-assigning / overriding it here directly again, is there a reason for that?

Reply:
you are right, is not needed. I've remove the first line